### ISSUE TITLE
Question regarding Async.Start

### ISSUE THREAD

**Issue by dbrattli** [fable-compiler/Fable#1826]
Currently the compiler will emit the warning "FABLE: Async.Start will behave as StartImmediate" which can be quite noisy when using e.g HRM. I don't want to change all my `Async.Start` to `Async.StartImmediate` since the code can also be used for .NET, and I also don't want to fill my code with `#if FABLE_COMPILER`. Is there a way to silence this warning using an attribute or something?

**Comment by dbrattli** [comment]
Made my own `Start'` that fixed the problem, so closing this issue:

```fs
    type Async with
        /// Starts the asynchronous computation in the thread pool, or
        /// immediately for Fable. Do not await its result. If no cancellation
        /// token is provided then the default cancellation token is used.
        static member Start' (computation:Async<unit>, ?cancellationToken: CancellationToken) : unit =
            #if FABLE_COMPILER
                Async.StartImmediate (computation, ?cancellationToken=cancellationToken)
            #else
                Async.Start (computation, ?cancellationToken=cancellationToken)
            #endif
```

**Comment by alfonsogarciacaro** [comment]
It's tricky, maybe we should detect when there are multiple warnings of the same type (though currently we're not assigning codes to Fable errors/warning) and only show the first one 🤔

**Comment by dbrattli** [comment]
Not sure that would help since with HRM you would still get the 1 message in your console every time you make a change to your application
